### PR TITLE
App promo: Fix reference to old site

### PR DESCRIPTION
**Pull request by jasmussen**
This PR updates the URL to both the image and the link for the desktop app download.

### CODE REVIEW

File: client/components/app-promo/index.jsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ export default React.createClass( {
 			return null;
 		}
 
-		var promo_link = 'https://desktop.wordpress.com/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
+		var promo_link = 'https://apps.wordpress.com/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
```
Comment (klimeryk) [inline client/components/app-promo/index.jsx]: It seems that the direct link to the desktop app is `https://apps.wordpress.com/desktop/` (and also this is the URL that `desktop.wordpress.com` redirects to).

**Comment by klimeryk** [comment]
There are also old links pointing to `desktop.wordpress.com` in https://github.com/Automattic/wp-calypso/blob/ab614064b3356e8ee0a1eda566de07e0f4af6cde/client/me/get-apps/index.jsx :)

**Comment by jasmussen** [comment]
Excellent feedback and great catches, @klimeryk! I've addressed your points. Let me know what you think.

**Comment by klimeryk** [comment]
Sweet! :bow:
:shipit:

**Pull request merged**